src/entity/user/UserStreak.ts

```diff
@@ -0,0 +1,23 @@
+import { Index, Column, PrimaryColumn, Entity } from 'typeorm';
+
+@Entity()
+export class UserStreak {
+  @PrimaryColumn({ type: 'text' })
+  @Index()
+  userId: string;
```

Comment:
You're missing a foreign key definition here

Reply:
Applied the definition 🥳